### PR TITLE
VIH-10839 Push additional event messages

### PR DESCRIPTION
**Pull request by oliver-scott**
### Jira link
https://tools.hmcts.net/jira/browse/VIH-10839

### Change description
Pushes some additional messages for hearing updates, to service the auto-refresh on the hearing list pages in video web

### CODE REVIEW

File: charts/vh-booking-queue-subscriber/values.dev.template.yaml [hunk 1]
```diff
@@ -3,13 +3,14 @@ java:
   image: '${IMAGE_NAME}'
   releaseNameOverride: ${RELEASE_NAME}
   environment:
-    QUEUENAME: booking
+    QUEUENAME: oliver-booking
```
Comment (oliver-scott) [inline charts/vh-booking-queue-subscriber/values.dev.template.yaml:6]: TODO revert before merge

File: BookingQueueSubscriber/BookingQueueSubscriber.Services/VideoWeb/IVideoWebService.cs [hunk 2]
```diff
@@ -11,5 +11,7 @@ public interface IVideoWebService
         Task PushUnlinkedParticipantFromEndpoint(Guid conferenceId, string participantUserName, string jvsEndpointName);
         Task PushLinkedNewParticipantToEndpoint(Guid conferenceId, string participantUserName, string jvsEndpointName);
         Task PushCloseConsultationBetweenEndpointAndParticipant(Guid conferenceId, string participantUserName, string jvsEndpointName);
+        Task PushHearingCancelledMessage(Guid conferenceId);
+        Task PushHearingDetailsUpdatedMessage(Guid conferenceId);
```
Comment (shaed-parkar) [inline BookingQueueSubscriber/BookingQueueSubscriber.Services/VideoWeb/IVideoWebService.cs:15]: Would it be worth providing a payload of the latest object?

Comment (shaed-parkar) [reply BookingQueueSubscriber/BookingQueueSubscriber.Services/VideoWeb/IVideoWebService.cs:15]: Actually, the video web could probably force get on the conference service, resulting in an up-to-date cache.

**Review by shaed-parkar — APPROVED** [review]
Looks good

**Review by komaldabhi1509 — APPROVED** [review]

**Pull request merged**